styles/globals.css

```diff
@@ -0,0 +1,14 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+    body {
```

Comment:
As you are using a new next.js version and folder structure, the body styles can be directly applied to the body in `_document` file